packages/tasks_panel/test/panels/task-editor-panel.spec.ts

```diff
@@ -120,6 +120,28 @@ describe("TaskEditorPanel class", () => {
       expect(panel2).to.exist;
       expect(panel2).to.not.eq(panel1);
     });
+
+    it("creates an instance on each instantiation, when it not the same", async () => {
```

Comment:
your test says: `creates an instance on each instantiation, when it not the same`

But the test seems to verify that two created instances are the **same** instance
if they are created with the same values.

With this description you should create two instances with different values and verify they are not the same instance (!===)

Reply:
rephrased